src/content/docs/en/guides/deploy/netlify.mdx

```diff
@@ -4,19 +4,19 @@ description: How to deploy your Astro site to the web on Netlify.
 type: deploy
 i18nReady: true
 ---
-[Netlify](https://netlify.com) offers hosting and serverless backend services for web applications and static websites. Any Astro site can be hosted on Netlify! 
+[Netlify](https://netlify.com) offers hosting and serverless backend services for web applications and static websites. Any Astro site can be hosted on Netlify!
 
 This guide includes instructions for deploying to Netlify through the website UI or Netlify's CLI.
 
 ## Project Configuration
 
-Your Astro project can be deployed to Netlify in three different ways: as a static site, a server-rendered site, or an (experimental) edge-rendered site.
+Your Astro project can be deployed to Netlify in three different ways: as a static site, a server-rendered site, or an edge-rendered site.
 
 ### Static Site
 
-Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Netlify. 
+Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Netlify.
 
-### Adapter for SSR/Edge
+### Adapter for SSR
 
 To enable SSR in your Astro project and deploy on Netlify:
```

Comment:
```suggestion
To enable SSR in your Astro project and deploy on Netlify, including using Netlify's edge functions:
```
It seems like really, this is the best bang for your buck re: letting people know that you'll use same the serverless adapter for both.